_posts/2019-01-30-lecture-05.md

```diff
@@ -0,0 +1,781 @@
+	
+	$$ \hat \mu_{k, MLE} = \text{argmax}_{\mu} l(\theta; D) \Rightarrow \hat\mu_{k, MLE} = \frac{\sum_n y_{n,k} x_n}{\sum_n y_{n,k}} = \frac{\sum_{n} y_{n,k} x_n}{n_k} $$, the average of samples of class $ k $
+
+#### Bayesian Estimation of Conditional Gaussian
+<figure>
+  <div class="row">
+    <div class="col one">
+      <img src="{{ '/assets/img/notes/lecture-05/gaussian_bayesian.png' | relative_url }}" />
+    </div>
+  </div>
+  <figcaption>
+    <strong>Bayesian Estimation of Conditional Gaussian</strong> 
+  </figcaption>
+</figure>
+- Prior:\\
+$$ 
+P(\pi | \alpha) = Dirichlet(\pi; \alpha)
+$$
+
+$$
+P(\mu_k | \nu) = \mathcal{N}(\mu_k; \nu, \tau^2)
+$$
+
+- Posterior mean:
+
+	$$ \pi_{k, Bayes} = \frac{N}{N + \vert a \vert_1} \hat \pi_{k, MLE} + \frac{\vert a \vert_1}{N + \vert a \vert_1} \frac{a_k}{ \vert a \vert_1} = \frac{n_k + a_k}{N + \vert a \vert_1} $$ where $ \vert a \vert_1 = \sum_{k=1}^K a_k $ 
+	
+	$$ \mu_{k, Bayes} = \frac{n_k/ \sigma^2}{n_k/\sigma^2 + 1/\tau^2}\hat{\mu}_{k, MLE} + \frac{1/\tau^2}{n_k/\sigma^2 + 1/\tau^2}\nu $$,
+	$$ \sigma_{Bayes}^{-2} = \frac{N}{\sigma^2} + \frac{1}{\tau^2} $$ 
+
+#### Gausian Distriminative Analysis:
+- Joint probability of a datum and its label is:
+	$$ P(x_n, y_{n,k} = 1 | \mu, \sigma ) = p(y_{n,k} = 1) p(x_n | y_{n,k} = 1, \mu, \sigma) = \pi_k \frac{1}{\sqrt{2\pi \sigma^2}}\exp\{-\frac{1}{2\sigma^2}(x_n - \mu_k)^2 \} $$
+
+- Predict the conditional probability of label given the datum:
+	$$ p(y_{n,k} = 1 | x_n, \mu, \sigma) = \frac{\pi_k (2\pi\sigma^2)^{-1/2} \exp \{ -(x_n - \mu_k)^2 /2\sigma^2 \} }{\sum_{k'} \pi_{k'} (2\pi\sigma^2)^{-1/2} \exp \{ -(x_n - \mu_{k'})^2 /2\sigma^2 \} } $$
+
+- Frequentist Approach: fit $ \pi $, $ \mu $ and $ \sigma $ from data first
+- Bayesian Approach: compute the posterior dist. of the parameters first
+
+### Linear Regression
+
+The data: $ \{(x_1, y_1), (x_2, y_2), \cdots, (x_N, y_N) \} $, where $ x $ is an input vector and $y$ is a response vector(either continuous or discrete)
+
+A regression scheme can be used to model $ p(y \vert x) $ directly rather than $ p(x,y) $
+
+#### A discrimintative probabilitstic model
+Assume $ y_i = \theta^T x_i + \epsilon_i $, where $\epsilon$ is an error term of unmodeled effects or random noise. Assume $ \epsilon_i \sim \mathcal{N}(0, \sigma^2) $, then 
+$$ 
+p(y_i | x_i; \theta) \sim \mathcal{N}(\theta^Tx_i, \sigma^2) = \frac{1}{\sqrt{2\pi\sigma^2}} \exp\{ -\frac{1}{2\sigma^2} (y_i - \theta^Tx_i)^2 \} 
+$$
+
+And by that each $(y_i, x_i$ are i.i.d, we have:
+$$ 
+L(\theta) = \prod_{i=1}^N p(y_i | x_i; \theta) = (\frac{1}{\sqrt{2\pi\sigma^2}})^N \exp\{ - \frac{1}{2\sigma^2}\sum_{i=1}^N(y_i - \theta^T x_i)^2 \}
+$$
+
+hence:
+$$ 
+l(\theta) = n\log \frac{1}{\sqrt{2\pi\sigma^2}} - \frac{1}{2\sigma^2}\sum_{i=1}^N(y_i - \theta^Tx_i)^2
+$$
+
+we notice that maximizing this term w.r.t $\theta$ is equivalent to minimizing a MSE(mean square error):
```

Comment:
is equivalent to minimizing a MSE(mean square error):
-> is equivalent to minimizing the MSE (mean square error):